CHANGELOG.md

```diff
@@ -2,7 +2,9 @@
 
 Note: We're not following semantic versioning yet, we are going to talk about this soon.
 
-## Unreleased
+## 0.0.32 (Breaking release)
+
+** This release changes the name of package. ** It's now published as `govuk-frontend` on `npm`.
```

Comment:
did you try to make this bold?

Reply:
I did! Markdown preview for Atom is more lenient than Github 😶